ui/src/components/Gene/Gene.tsx

```diff
@@ -7,31 +7,69 @@ import GeneDrugs from "./GeneDrugs";
 import GeneLocusZoom from "./GeneLocusZoom";
 import GeneContextProvider, { GeneContext, GeneState } from "./GeneContext";
 import GeneBanner from "./GeneBanner";
-import GenePqtls from "./GenePqtlColocalization"
 import { ConfigurationWindow } from "../Configuration/configurationModel";
-import { GeneParams } from "./geneModel";
+import { GeneParams, Gene as GeneModel } from "./geneModel";
 import { RouteComponentProps } from "react-router-dom";
 import { hasError, isLoading } from "../../common/CommonLoading";
+import GenePqtlColocsTab from "./GenePqtlColocalizationTab";
 
 declare let window: ConfigurationWindow;
 const { config } = window;
 const showLOF : boolean = config?.userInterface?.gene?.lossOfFunction != null;
 const showPqtl : boolean = config?.userInterface?.gene?.pqtlColocalizations != null;
+const showGeneColocs : boolean = config?.userInterface?.gene?.geneColocalizations != null;
```

Comment:
can you add example of these configurations into public config...

Reply:
added